coda/coda_mdstore/factories.py

```diff
@@ -66,6 +66,50 @@ class FullBagFactory(BagFactory):
     )
 
 
+class OAIBagFactory(BagFactory):
+    """
+    Factory to create a Bag object with two related Bag_Info objects, and
```

Comment:
It _is_ creating a `Bag` with 4 related `Bag_Info` objects, specifically for testing the OAI functions.